dof/src/dof.rs

```diff
@@ -56,7 +56,7 @@ pub enum Error {
 }
 
 /// Represents the DTrace data model, e.g. the pointer width of the platform
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone, Copy, serde::Serialize)]
```

Comment:
any reason you're not doing `use serde::Serialize` ?

Reply:
Nope, I can do that.